config/base.json

```diff
@@ -1,3 +1,4 @@
 {
-  "generate_icicle": false
+  "generate_icicle": false,
+  "oz_overrides" : "{'libvirt': {'memory': 2048}}"
```

Comment:
This increases the memory of the VM that is used by ImageFactory to create the image to from 1GB to 2GB. I did this for safety reasons since we're using a couple of tmpfs file systems in that VM. I don't have the total amount of space that they use but it is less that 1GB.